.github/PULL_REQUEST_TEMPLATE/add_or_update_translation.md

```diff
@@ -0,0 +1,58 @@
+---
+name: Add or update a translation
+about: Make Fly-Pie international! 🌐
+title: '<Add/Update> <LANGUAGE> translation'
+labels: translation
+assignees: ''
+---
+
+<!--
+Thank you for contributing a translation to Fly-Pie!
+This template will help you to to provide a high quality contributon to the project.
+Please replace words in pointy <brackets> with suitable replacements.
+
+---
+
+This block of text is a comment - it will not show up in the final version of the
+Pull request, but provide additional info to you on how to do things correctly.
+You should not delete them, in case you would like to look something up later.
+In order to do so once you submitted the PR, click on "edit" in the top right corner.
+-->
+
+## Requirements to get this PR accepted
+
+<!--
+Make sure to comply to all requirements and tick the boxes once you have done the task.
+Look at existing translations for reference. Even if your PR doesn't meet all requirements
+right now, please do submit it anyways! We are a friendly community and eager to help you out.
+
+Once all boxes are ticked, your PR is ready to be merged (providing Fly-Pie's maintainer
+is ok with it - but I'm sure they will be! xD)
+-->
+
+- [ ] I followed the "Translating FlyPie" section of the README.
+- [ ] I complied to the Contributing Guidelines. <!--see CONTRIBUTING.md-->
+- [ ] I made the following changes to my `<LANGUAGE>.po` file: <!--You'll find most of them in the first code block, around lines 1-20-ish.-->
+  - [ ] Fill in the `<LANGUAGE>`.
+  - [ ] In case you are the first translator, fill in your info and the year. If not, just tick this box anyway.
+  - [ ] Set `PO_Revision-Date` to an appropriate point in time.
+        <!--It doesn't have to be accurate to the second, but at least the hour should be accurate.-->
+  - [ ] Set `Last-Translator` accordingly.
+        <!--Format: Firstname Lastname <my@email.address>.
+        Alternatively, use your GitHub handle.-->
+  - [ ] Update `Language-Team`. If your name is already on there, just tick this box anyway.
+        <!--Please place your full name or GitHub handle at the end of the comma-seperated list.-->
+  - [ ] Set `Language` to the language code (`de_DE`, `en_GB`) or just the country code (`de`, `en`).
+        <!--If there are special words that are written differently in two countries with the same
+        language (e. g. "color" and "colour" in English), the first variant is obligatory.-->
+  - [ ] Make sure that `charset` is set to `UTF-8`.
+        <!--This allows for special characters like ä, ö, ü, è etc. to be displayed correctly.-->
+
+## Additional ideas/questions
+
```

Comment:
Maybe we could ask people here if they want to get notified when the translation needs to be updated?